src/main/scala/generate/database/PostgresGenerator.scala

```diff
@@ -0,0 +1,30 @@
+package generate.database
+
+import generate.database.ast._
+
+object PostgresGenerator extends DatabaseGenerator {
+
+  private def parseColumn(column: Column): String =
+    column match {
+      case IntColumn(name)        => s"    $name INT"
```

Comment:
Not so much a fan of this way of doing indentation, feels like it's very dependent on the raw strings.
Could we have a method that takes an argument and indents the provided text by the provided length?

```scala
def indent(val: String, length: Int = 2): String = { ... }
```

Reply:
We've made this somewhere too, so please could you add this to a Utils/StringUtils object?

```scala
// indent a multi-line string: insert two spaces at the start of the string and immediately following newline chars
def indentString(str: String, length: Int = 2): String = str.replaceAll("^|(?<=\n)", " " * length)
```